user.go

```diff
@@ -20,9 +20,9 @@ type User struct {
 
 // Officership represents an officership a user holds.
 type Officership struct {
-	OfficerId   uint   `json:"officerid,string"`
+	OfficerID   uint   `json:"officerid,string"`
```

Comment:
It seems like an odd inconsistency that we use `ID` in some places and `NameOfThingID` in others.  In this case it makes sense because this is a bridge table, but…